### PR TITLE
css-three-column

### PR DESCRIPTION
**Pull request by aklepper20**
three-column-layout | klepper

image

### CODE REVIEW

**Comment by aklepper20** [comment]
I believe I am missing a div surrounding the column-third that would pull together the 3 content items inside

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  margin-top: 18px;
+}
+
+.logo-border {
+  width: 28%;
+  height: 45%;
+  text-align: center;
+  background-color: white;
+  border-radius: 60%;
+  margin-left: 33%;
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css:52]: That's a lot of margin! What is the purpose of this? If it is to center the image in its container, there are better ways to accomplish this, such as `margin: auto` and flexbox

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!!

File: css-three-column-layout/styles.css (hunk 1 above)
Comment (aklepper20) [reply css-three-column-layout/styles.css]: image
margin: auto in da house

**Review by codingcodymiller — APPROVED** [review]
Nice job!!!

**Pull request merged**